processing/src/main/java/io/druid/query/aggregation/DoubleMaxAggregatorFactory.java

```diff
@@ -51,25 +53,44 @@ public DoubleMaxAggregatorFactory(String name, String fieldName)
   }
 
   @Override
-  public Aggregator factorize(ColumnSelectorFactory metricFactory)
+  protected BaseDoubleColumnValueSelector selector(ColumnSelectorFactory metricFactory)
   {
-    return new DoubleMaxAggregator(getDoubleColumnSelector(metricFactory, Double.NEGATIVE_INFINITY));
+    return getDoubleColumnSelector(
+        metricFactory,
+        Double.NEGATIVE_INFINITY
```

Comment:
I believe that's because we use this as default when aggregating, No plan to change this, nullability is handled in NullableAggregatorFactory which encapsulates this.

Reply:
Ok, thanks.